core/src/main/java/com/scalar/db/transaction/consensuscommit/CrudHandler.java

```diff
@@ -273,16 +288,17 @@ Optional<TransactionResult> getFromStorage(Get get) throws CrudException {
 
   private Scanner scanFromStorage(Scan scan) throws CrudException {
     try {
-      scan.clearProjections();
+      BuildableScanOrScanAllFromExisting builder = Scan.newBuilder(scan);
```

Comment:
We needed to use a copy of the scan operation here. (Now, I think we might return different results for the same scan operations by changing the operation directly.)